sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionTest.java

```diff
@@ -122,7 +120,7 @@ public void ethereumLegacyEIP155TransactionTest() {
         // - to address: 0xd830603264bd3118cf95f1fc623749337342f9e9
         // - value: 3000000000000000000
         // - chain id: 1997
-        String metamaskHexStr = "de01f86d02843b9aca0082520894d830603264bd3118cf95f1fc623749337342f9e98829a2241af62c000080820fbea0f638802002d7c0a3115716f7d24d646452d598050ffc2d6892ba0ed88aeb76bea01dd5a7fb9ea0ec2a414dbb93b09b3e716a3cd05c1e333d40622c63d0c34e3d35";
+        String metamaskHexStr = "f86d02843b9aca0082520894d830603264bd3118cf95f1fc623749337342f9e98829a2241af62c000080820fbea0f638802002d7c0a3115716f7d24d646452d598050ffc2d6892ba0ed88aeb76bea01dd5a7fb9ea0ec2a414dbb93b09b3e716a3cd05c1e333d40622c63d0c34e3d35";
```

Comment:
just to double check.
Here we removed the encoded length of the raw tx data, right?

Reply:
Yes correct